outofcore/src/cJSON.cpp

```diff
@@ -349,7 +352,7 @@ static char *print_array(cJSON *item,int depth,int fmt)
 	/* Allocate an array to hold the values for each */
 	entries=static_cast<char**>(cJSON_malloc(numentries*sizeof(char*)));
 	if (!entries) return nullptr;
-	memset(entries,0,numentries*sizeof(char*));
+  std::fill_n(entries, numentries, nullptr);
```

Comment:
Same here: `*entries = Class{};` and in other places in this file

Reply:
This file could really use a `clang-format` pass, but I resisted.